backend/src/api.ts

```diff
@@ -315,6 +321,20 @@ loginCheckedPut('/coffee-chat', async (req, user) => ({
   coffeeChats: await updateCoffeeChat(req.body, user)
 }));
 
+// Coffee Chat Proof Image
```

Comment:
What if we unified this on the backend as well? So just have a generic `/image/:name(*)` (get), `/image-signed-url/:name(*)` (get), `/image/:name(*)` (delete) endpoints? I think the logic is the same across all the different use-cases anyways.

Instead of the utils, we'd have just three endpoints total (as opposed to 9)

Reply:
You should keep the image names the same though, so I would still make sure the images, when they're created, are prefixed correctly (i.e. `eventProofs/`, `coffeeChatProofs/`, `member-image/`)

Reply:
Oops, I missed unifying this file as well. Will do that!

Reply:
Even better if it's just prefixed with `/files` to be super generic :D